_pages/privacy.md

```diff
@@ -36,7 +38,7 @@ toc_sticky: true
 ## Event registration
 
 - We use a Google form to submit to the topic bazaar, but this is also stated
-  separately [here](https://rse-leaders.github.io/SORSE20/programme/bazaar/#submit).
+  separately [here]({% include fix-link.html link=submit_topic %}).
```

Comment:
Just to check - I saw this twice in the repo - did we get both of them?

Reply:
I only fixed one of them for which I got the error. Where is the other?

Reply:
looking for `rse-leaders` within all repository files did not show anything